src/main/webapp/css/style.css

```diff
@@ -10,7 +10,7 @@
     padding: 20px;
     text-align:center;
     text-decoration: none;
-    font-size: 50px;
+    font-size: 30px;
```

Comment:
This fix is good for now but it would be ideal for the text to resize based on the button size